From 45597a58e98f351b18db8444292b1cf6dd0cd810 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Sat, 9 Dec 2023 23:34:08 -0800
Subject: [PATCH] reexport.h: Include unistd.h to compile with musl

Fixed error when compile with musl
reexport.c: In function 'reexpdb_init':
reexport.c:62:17: error: implicit declaration of function 'sleep' [-Werror=implicit-function-declaration]
   62 |                 sleep(1);


Upstream-Status: Submitted [https://marc.info/?l=linux-nfs&m=170254661824522&w=2]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 support/reexport/reexport.h | 1 +
 1 files changed, 1 insertions(+)

diff --git a/support/reexport/reexport.h b/support/reexport/reexport.h
index 85fd59c..02f8684 100644
--- a/support/reexport/reexport.h
+++ b/support/reexport/reexport.h
@@ -1,6 +1,8 @@
 #ifndef REEXPORT_H
 #define REEXPORT_H
 
+#include <unistd.h>
+
 #include "nfslib.h"
 
 enum {
-- 
2.42.0

